cloudbuild.yaml

```diff
@@ -0,0 +1,4 @@
+steps:
+- name: 'hashicorp/terraform:light'
```

Comment:
Actually one issue - can we do this with our Exekube image? That way the TF version running the fmt check will be consistent with the execution environment, especially as we don't have a version tag or anything pinning the version used (what if they release 0.12 new HCL?)

Reply:
Yes, we can. One downside of that is that our exekube image is pretty heavy, which increases build times and also causes some extra charges (normally cloud build is free for open repos). We may want to push exekube image to Google Container Registry to improve this.